src/renderer/modals/Send/Body.js

```diff
@@ -193,6 +200,44 @@ const Body = ({
     [account, setAccount],
   );
 
+  const handleChangeNFT = useCallback(
+    nextNft => {
+      const nextAccount = accounts.find(a => a?.nfts.find(({ id }) => nextNft.id));
+
+      if (account !== nextAccount) {
+        setAccount(nextAccount);
+      }
+      const bridge = getAccountBridge(nextAccount);
+      const standard = nextNft.collection.standard.toLowerCase();
+
+      setTransaction(
+        bridge.updateTransaction(transaction, {
+          tokenIds: [nextNft.tokenId],
+          quantities: standard === "erc1155" ? [BigNumber(1)] : undefined,
```

Comment:
this logic should be handled in the transaction module for `erc1155.transfer` on live-common you'll need to duplicate this on mobile if you keep it here.

Reply:
agreed. fundamentally i'm not even sure it's a good idea that we have the knowledge, on LLD/LLM side, of "erc1155", it should be abstracted out to "nft" because it's now it looks like in the model.
but too late to change this right now.

Reply:
Yep I guess Juan is correct, we will always need this distinction in the LLD/LLM as ERC721 and ERC1155 have a different flow. We could probably skip the ternary though because the `erc721.transfer` module doesn't even use the quantities property anyway 🤷

Reply:
I removed the ternary, I'll add it again if I notice a bug once we tackle the send feature tests.